onnxruntime/core/session/ort_apis.h

```diff
@@ -533,4 +533,25 @@ ORT_API_STATUS_IMPL(RunOptionsAddActiveLoraAdapter, _Inout_ OrtRunOptions* optio
 
 ORT_API_STATUS_IMPL(SetEpDynamicOptions, _Inout_ OrtSession* sess, _In_reads_(kv_len) const char* const* keys,
                     _In_reads_(kv_len) const char* const* values, _In_ size_t kv_len);
+
+ORT_API(const OrtModelBuilderApi*, GetModelBuilderApi);
+
+ORT_API_STATUS_IMPL(CreateTensorWithDataAndDeleterAsOrtValue, _In_ OrtAllocator* deleter,
+                    _In_ void* p_data, size_t p_data_len,
```

Comment:
Would `p_data` be written by ORT? Or should it take `const void*` if it is only read by ORT? A similar question is for `CreateTensorWithDataAsOrtValue()` which takes `_Inout_ void* p_data`.

https://github.com/microsoft/onnxruntime/blob/skottmckay/ModelBuilderAPI_PR/include/onnxruntime/core/session/onnxruntime_c_api.h#L1359C83-L1359C103